egs/csj/s5/local/chain/tuning/run_tdnn_7k.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=8
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=false
```

Comment:
I think speed_perturb should default to true.  We always do it like this.  If you're concerned about speed you can reduce the number of epochs slightly.